x-pack/docs/en/security/authentication/security-domain.asciidoc

```diff
@@ -0,0 +1,97 @@
+[role="xpack"]
+[[security-domain]]
+=== Security domains
+
+beta::[]
+
+Security domains are a method of grouping <<realms,realms>> to enable resource
+sharing across different realms for the same user.
+
+==== How security domain works
+
+Users can create various resources in {es} that they own, access, and manage.
+For example, using the <<async-search,async search API>> to create a search
+request, <<security-api-create-api-key,creating an API key>>, or activating a
+<<user-profile,user profile>>.
```

Comment:
Strictly, users don't access or manage their own profiles (nor can they "create" them)

Perhaps we should say
```suggestion
Some types of resources in {es} are owned by a single user.
For example <<async-search,async search contexts>>,
<<security-api-create-api-key,API keys>>,
and <<user-profile,user profiles>> each have an owning user.
```

Reply:
Good catch!